### PR TITLE
Add cluster status enums to Swagger

### PR DESCRIPTION
**Pull request by helgridly**
It doesn't look great in the Swagger UI rendering but it's there now. On request of the AoU folks.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -329,6 +339,7 @@ definitions:
         description: Google's operation ID for the cluster
       status:
         type: string
+        enum: *CLUSTERSTATUS
```
Comment (rtitle) [inline src/main/resources/swagger/api-docs.yaml]: What syntax is this? C reference/pointer?

Comment (helgridly) [reply src/main/resources/swagger/api-docs.yaml]: It's a YAML anchor: https://swagger.io/docs/specification/2-0/enums/

Comment (helgridly) [reply src/main/resources/swagger/api-docs.yaml]: actually, i wonder if the merge key syntax will be less gross...

Comment (helgridly) [reply src/main/resources/swagger/api-docs.yaml]: According to the online swagger editor, no. Hmph.

**Pull request merged**